bundles/org.openhab.binding.automower/src/main/java/org/openhab/binding/automower/internal/rest/api/automowerconnect/AutomowerConnectApi.java

```diff
@@ -62,22 +73,120 @@ public MowerResult getMower(String appKey, String token, String mowerId) throws
         request.method(HttpMethod.GET);
 
         ContentResponse response = executeRequest(appKey, token, request);
-
+        // logger.trace("getMower: {}", response.getContentAsString());
```

Comment:
```suggestion
```

Reply:
removed the comment. json data received is now logged to logged to trace.